Content.Server/Chemistry/ReagentEffects/PlantMetabolism/RobustHarvest.cs

```diff
@@ -11,14 +11,11 @@ namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
     [DataDefinition]
     public sealed partial class RobustHarvest : ReagentEffect
     {
-        [DataField]
-        public int PotencyLimit = 50;
+        [DataField] static int PotencyLimit = 50;
 
-        [DataField]
-        public int PotencyIncrease = 3;
+        [DataField] static int PotencyIncrease = 3;
 
-        [DataField]
-        public int PotencySeedlessThreshold = 30;
+        [DataField] static int PotencySeedlessThreshold = 30;
```

Comment:
i think it would make more sense to just make these private. Just make sure you add strings specifiers to the datafield attributes (i.e. `[DataField("potencyLimit")]` ) so as to not break compatibility.

Reply:
Maybe I'm misunderstanding how `static` and the RobustHarvest class works. I changed this so `PotencyLimit` could be referenced outside of the class (for #25575), since it is always 50 in the game and chemicals do not change function. Would other forks not be able to change `PotencyLimit` in this file?

Reply:
wont making the fields static mean that its not possible for it to be set in yml to begin with

Reply:
Hm, if forks change these values with YAML instead of changing the numbers in the files, maybe I should scrap this and find another solution that doesn't involve referencing variables in this file.

Reply:
this whole pr is just hardcoding it, remove datafield if they are all constants not meant to be changed

OOOOOR ideally make the effect not hyper specific, and more granular so it can be reused easily

Reply:
I thought the chem was a constant not meant to be changed, but it sounds like there are forks that change this value using YAML that do not want to make this chem standard. I'll scrap it.